jobs/transforms/base_model_audits.py

```diff
@@ -251,6 +252,58 @@ def process(self, input_model):
             yield audit_errors.ModelDomainObjectValidateError(input_model, e)
 
 
+class ValidateCommitCmdsSchema(beam.DoFn):
```

Comment:
```suggestion
class BaseValidateCommitCmdsSchema(beam.DoFn):
```
And add a comment stating that the decorator is not required. Thanks.

Reply:
@rohitkatlaa done PTAL!